es-util.sh

```diff
@@ -133,14 +135,14 @@ log "Getting bindings for ${SERVICE_NAME}."
 SVC_JSON=$(cf curl \
   "/v2/spaces/$(cat ~/.cf/config.json | jq -rc .SpaceFields.Guid)/service_instances?return_user_provided_service_instances=true&q=name%3A${SERVICE_NAME}&inline-relations-depth=1")
 
-ACCESS_KEY=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.access_key | select(. != null)')
-SECRET_KEY=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.secret_key | select(. != null)')
-BUCKET=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.bucket | select(. != null)')
+ACCESS_KEY=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.access_key | select(. != null)' | head -n1 )
```

Comment:
(and I tested it against `dev`, which has the two service bindings that were causing the problem, and it seemed to work)